### PR TITLE
added tests

### LINKED ISSUE #103: (SP: 1) Write unit test for "InputWithIcon" component
### Component unit test
Unit test for "**InputWithIcon**" component
Scenaries descriptions:
- [x] Check if the "**start**" icon that was passed in props should be in the document 
- [x] If the value of input is not empty it should show "**clear**" icon

[Link to component](https://github.com/Space2Study-Project/Space2Study-FrontEnd/blob/develop/src/components/input-with-icon/InputWithIcon.jsx)

Current coverage:

Image

### PR DESCRIPTION
**Pull request by Mygal-Alex**
image

image

### CODE REVIEW

**Review by AndriiNasadiuk — APPROVED** [review]
ok

**Review by ArtemHolikov — APPROVED** [review]
Nice tests

**Review by marianamartsy — APPROVED** [review]
god job
ps. make, please, the pull request title more understandable than it currently is 👍

**Review by LeonidShamarin — APPROVED** [review]
Ok

File: tests/unit/components/input-with-icon/InputWithIcon.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { screen, render } from '@testing-library/react'
+import InputWithIcon from '~/components/input-with-icon/InputWithIcon'
+import CloudUploadIcon from '@mui/icons-material/CloudUpload'
+
+describe('Input With Icon component', () => {
+  const props = {
+    value: 'TEST',
+    startIcon: <CloudUploadIcon /> // random test icon
```
Comment (vlad-khrychov) [inline tests/unit/components/input-with-icon/InputWithIcon.spec.jsx]:
```suggestion
    startIcon: <CloudUploadIcon />
```
What about the comment rule in this project? I think they are forbidden.

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

File: tests/unit/components/input-with-icon/InputWithIcon.spec.jsx (hunk 1 above)
Comment (Mygal-Alex) [reply tests/unit/components/input-with-icon/InputWithIcon.spec.jsx]: Yes, I'm sorry
I left a comment to explain the solution. 
and I'm deleting it now.

**Pull request merged**